configuration/src/gas.rs

```diff
@@ -0,0 +1,99 @@
+//! Per-chain gas configurations
```

Comment:
Should any of these be optional? or omitted from the configs?

Reply:
Passing in config for chain is optional. If you don't pass in anything, agent RPC resorts to `eth_estimateGas`